workflow/scripts/build_primer_regions.py

```diff
@@ -0,0 +1,35 @@
+import pandas as pd
+
+chunksize = 10 ** 6
+with open(snakemake.output[0], "w") as out:
+    with open(snakemake.log[0], "w") as log_file:
+        for data_primers in pd.read_csv(
+            snakemake.input[0],
+            sep="\t",
+            header=None,
+            chunksize=chunksize,
+            usecols=[0, 1, 2, 3, 4, 5],
+        ):
+            valid_primers = data_primers[0] == data_primers[3]
+            valid_data = data_primers[valid_primers]
+            valid_data.iloc[:, [1, 4]] += 1
```

Comment:
This line gives me an headache.
There always pops up a [SettingWithCopyWarning](https://www.dataquest.io/blog/settingwithcopywarning/) and I get what's the issue here.
But I don't find a way on how two return a view of both columns instead of a copy.
Any suggestion how to solve this?

Reply:
```suggestion
            valid_data.iloc[:, 1] += 1
            valid_data.iloc[:, 4] += 1
```

Reply:
Does that solve it?

Reply:
Unfortunately not. The warning stays the same.